modules/claims_api/app/swagger/claims_api/forms/form_0966_v1_example.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "form/0966",
+  "attributes": {
+    "type": "compensation",
+    "participant_claimant_id": 123456789,
+    "participant_vet_id": 987654321,
```

Comment:
same question as above

Reply:
it should be fine for example purposes.

Reply:
we want this to be an integer after all.